README.md

```diff
@@ -7,16 +7,49 @@
 
 ![openexr](/OpenEXR/doc/images/windowExample1.png)
 
-**OpenEXR** is a high dynamic-range (HDR) image file format for use in
-computer imaging applications. It supports stereoscopic and deep
-images. OpenEXR's features include:
-
-* Higher dynamic range and color precision than existing 8- and 10-bit
-  image file formats.
+The OpenEXR project provides the specification and reference
+implementation of the EXR file format, the professional-grade image
+storage format of the motion picture industry. The purpose of format
+is to accurately and efficiently represent high-dynamic-range
+scene-linear image data and associated metadata, with strong support
+for multi-part, multi-channel use cases. The library is widely used in
+host application software where accuracy is critical, such as
+photorealistic rendering, texture access, image compositing, deep
+compositing, and DI.
+
+## OpenEXR Mission Statement
+
+The goal of the OpenEXR project is to keep the format reliable and
+modern and to maintain its place as the preferred image format for
+entertainment content creation. Major revisions are infrequent, and
+new features will be carefully weighed against increased complexity.
+
+The principal priorities of the project are:
+* Robustness, reliability, security
+* Backwards compatibility, data longevity
+* Performance - read/write/compression/decompression time
+* Simplicity, ease of use, maintainability
+* Wide adoption, multi-platform support - Linux, Windows, macOS, and others
+
+OpenEXR is intended solely for 2D data. It is not appropriate for
+storage of volumetric data, cached or lit 3D scenes, or more complex
+3D data such as light fields.
+
+Also a part of OpenEXR, the IlmBase library is a basic, light-weight,
+and efficient representation of 2D and 3D vectors and matrices and
```

Comment:
Due to the extensiveness of the test suite for Imath, I'd be happy/like to suggest, adding "correct" to the descriptors. (When writing non-math based math code, I often implement the correctness tests using Imath as a ground truth.)